install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
+
+set -e
+
+TMP_DIR=/tmp/migrate-k8s
+LOG_LEVEL=info
+PRIVATE_ADDRESS=
+PUBLIC_ADDRESS=
+AIRGAP=0
+# If all state needed from the native app has been saved we can skip some steps
+HAS_NATIVE_STATE=0
+HAS_KUBERNETES=0
+HAS_APP=0
+
+{% include 'common/cli-script.sh' %}
+{% include 'common/common.sh' %}
+{% include 'common/log.sh' %}
+{% include 'common/system.sh' %}
+
+startNativeScheduler() {
+    logStep "Ensuring native scheduler is running"
+
+    systemctl start replicated
+    systemctl start replicated-operator
+    systemctl start replicated-ui
+
+    # ensure replicatedctl wrapper is for native
+    if ! grep -q "sudo docker exec" "/usr/local/bin/replicatedctl" 2>/dev/null ; then
+        installCliFile "sudo docker exec" "replicated"
+    fi
+
+    waitReplicatedctlReady
+
+    logSuccess "Native scheduler is running"
+}
+
+waitReplicatedctlReady() {
+    for i in {1..30}; do
+        if isReplicatedctlReady; then
+            return 0
+        fi
+        sleep 2
+    done
+    return 1
+}
+
+isReplicatedctlReady() {
+    replicatedctl system status 2>/dev/null | grep -q '"ready"'
+}
+
+getNativeEnv() {
+    cat "$TMP_DIR/native.env" | grep "$1" | sed 's/=/ /g' | awk '{ print $2 }'
+}
+
+getK8sInitScript() {
+    getUrlCmd
+    if [ "$AIRGAP" != "1" ]; then
+        $URLGET_CMD "{{ replicated_install_url }}/{{ kubernetes_init_path }}?{{ kubernetes_init_query }}" \
+            > /tmp/kubernetes-init.sh
+    else
+        cp kubernetes-init.sh /tmp/kubernetes-init.sh
+    fi
+}
+
+stopNativeScheduler() {
+    logStep "Stopping native scheduler"
+    systemctl stop replicated
```

Comment:
Also maybe disable services and remove respective service files

Reply:
I'll be sure to add that, maybe after the K8s install is successful